app/include/zmk/split/bluetooth/central.h

```diff
@@ -5,4 +5,6 @@
 #include <zmk/behavior.h>
 
 int zmk_split_bt_invoke_behavior(uint8_t source, struct zmk_behavior_binding *binding,
-                                 struct zmk_behavior_binding_event event, bool state);
\ No newline at end of file
+                                 struct zmk_behavior_binding_event event, bool state);
+
+int zmk_run_behavior(struct zmk_behavior_binding *binding, struct zmk_behavior_binding_event event,uint8_t source,bool pressed);
```

Comment:
I think this general functionality is useful, but a few things:

* This should be in `keymap.h`, not this header.
* I don't love the name. We are triggering the pressed/released callbacks, not "running" something.